lib/commands/new_command.ts

```diff
@@ -25,6 +25,7 @@ export async function runNewCommand() {
 members = [
   "rs_lib",
 ]
+resolver = "2"
```

Comment:
Maybe we should just change this to `edition = "2021"`?

```suggestion
edition = "2021"
```

https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html#summary